Classes/Controller/ContentController.php

```diff
@@ -52,6 +55,27 @@ public function renderAction()
             );
         $flexform->setTyposcriptDefaults($this->settings['defaults']);
 
+        if (empty(str_replace("\r\n", '', $contentElement['bodytext']))) {
+            $file = $flexform->getCFile();
+            $linkService = GeneralUtility::makeInstance(LinkService::class);
+            $data = $linkService->resolveByStringRepresentation($file);
+            $content = '';
+            if ($data['type'] === 'url') {
+                $content = file_get_contents($data['url']);
```

Comment:
Question: Don't we need to make sure it's a local URL before retrieving the content and paste it into our page? Might be a potential security issue.

What do you think?